src/utils/filplus-edition.ts

```diff
@@ -1,25 +1,84 @@
-export type FilPlusEdition = {
+import { BadRequestException } from '@nestjs/common';
+import { DateTime } from 'luxon';
+
+export const DEFAULT_FILPLUS_EDITION_ID = 6;
+
+export type FilplusEditionConfig = {
   id: number;
-  start: number;
-  end?: number;
+  startTimestamp: number;
+  endTimestamp: number;
   lowReplicaThreshold: number;
   highReplicaThreshold: number;
 };
 
+export type FilPlusEdition = {
+  startDate: Date;
+  endDate: Date;
+  isCurrent: boolean;
+} & FilplusEditionConfig;
+
 // prettier-ignore
-const filPlusEditions: FilPlusEdition[] = [
-  { id: 5, start: 0, end: 1751846399, lowReplicaThreshold: 4, highReplicaThreshold: 10 }, // < 2026-07-06
-  { id: 6, start: 1751846400, lowReplicaThreshold: 4, highReplicaThreshold: 8 }, // >= 2026-07-07
+const filPlusEditions: FilplusEditionConfig[] = [
+  { id: 5, startTimestamp: 0, endTimestamp: 1751846399, lowReplicaThreshold: 4, highReplicaThreshold: 10 }, // < 2026-07-06 and before 2026-07-07
```

Comment:
`2026-07-06 and before 2026-07-07` what does it even mean. Also, the timestamps are from month 07 but edition 6 starts in month 06. Also, comments says 2026.